.github/workflows/publish.yml

```diff
@@ -154,3 +154,10 @@ jobs:
           command: "V=1 LINUXKIT_PKG_TARGET=manifest pkgs"
           dockerhub-token: ${{ secrets.RELEASE_DOCKERHUB_TOKEN }}
           dockerhub-account: ${{ secrets.RELEASE_DOCKERHUB_ACCOUNT }}
+
+  trigger_assets:
+    if: ${{ (startsWith(github.ref, 'refs/tags/')) && (github.event.repository.full_name == 'lf-edge/eve') }}
```

Comment:
Why do we have restriction on repository full name here?

Reply:
Do avoid triggering for a fork of the repo, I assume.

Reply:
But how would you test that workflow is working correctly before merging? My approach to it would be running workflow in your own fork